plugins/exporter.koplugin/target/json.lua

```diff
@@ -1,19 +1,46 @@
 local Device = require("device")
 local rapidjson = require("rapidjson")
+local _ = require("gettext")
+local md5 = require("ffi/MD5")
 
 -- json exporter
 local JsonExporter = require("base"):new {
     name = "json",
     shareable = Device:canShareText(),
 }
 
-local function format(booknotes)
+function JsonExporter:getMenuTable()
+    local menu = {
```

Comment:
nit: no need to store the table in a variable if the next thing we're going to do is return it.

you can return a table directly